src/gql/server/Buttons.tsx

```diff
@@ -178,57 +209,53 @@ export function Slew({ label, disabled, className }: { label: string; disabled:
     <MutationButton
       mutation={SLEW_MUTATION}
       variables={variables}
-      className={className}
-      label={label}
-      disabled={disabled || !selectedTarget.id || loading}
+      {...props}
+      loading={props.loading || loading}
+      disabled={props.disabled || !selectedTarget.id}
     />
   );
 }
 
-// OIWFS
-const OIWFS_MUTATION = graphql(`
-  mutation setOiwfsTarget(
-    $id: TargetId!
-    $name: NonEmptyString!
-    $ra: HmsString
-    $dec: DmsString
-    $epoch: EpochString
-    $wavelength: PosBigDecimal
-  ) {
-    oiwfsTarget(
-      target: {
-        id: $id
-        name: $name
-        sidereal: { ra: { hms: $ra }, dec: { dms: $dec }, epoch: $epoch }
-        wavelength: { nanometers: $wavelength }
-      }
-    ) {
-      result
-    }
-  }
-`);
+// TODO: is this needed anywhere? I'm not sure what it is
```

Comment:
Is this correct?

Reply:
That `OIWFS_MUTATION` is not used anymore, Javier added that logic to send parts of the configuration to the TCC, but as far as I remember, everything was changed to send a single block on the slew mutation.